X/XGBoost/build_tarballs.jl

```diff
@@ -37,9 +57,21 @@ fi
 install_license LICENSE
 """
 
+
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = expand_cxxstring_abis(supported_platforms())
+cuda_platforms = expand_cxxstring_abis(Platform("x86_64", "linux"))
+
+cuda_versions = [v"11.0", v"11.8"]
```

Comment:
Do you need a separate build for 11.8? If XGBoost doesn't have any specific code targeting it, its minimum requirement of 11.0 should be enough.

Reply:
It is not needed. I have removed the extra CUDA builds.